### PR TITLE
Delegated Account

### PR DESCRIPTION
**Pull request by madhavmansuriya40**

### CODE REVIEW

File: src/app/core/services/org-user.service.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export class OrgUserService {
     return this.apiService.post('/users', user);
   }
 
+  // TODO: move to v2
```
Comment (Julias0) [inline src/app/core/services/org-user.service.ts:29]: dei this is a post call

**Review by Julias0 — APPROVED** [review]

File: src/app/app.module.ts [hunk 2]
```diff
@@ -15,17 +15,22 @@ import { HttpConfigInterceptor } from './core/interceptors/httpInterceptor';
...
 import { InAppBrowser } from '@ionic-native/in-app-browser/ngx';
 
+import { DelegatedAccMessageComponent } from './shared/components/delegated-acc-message/delegated-acc-message.component';
+import  { SharedModule } from './shared/shared.module';
+
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    DelegatedAccMessageComponent
```
Comment (Julias0) [inline src/app/app.module.ts:24]: Why is this here? This should be declared inside the shared.module.ts

File: src/app/fyle/delegated-accounts/delegated-accounts.page.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          <ion-col size="2">
+            <div class="delegated--org-thumb">
+              {{ acc.us.full_name | initials | uppercase }}
+            </div>
+          </ion-col>
+          <ion-col size="10">
+            <div class="delegated--org-element-name">
+              {{acc.us.full_name}}
+            </div>
+            <div class="delegated--org-element-usdetails">
```
Comment (Julias0) [inline src/app/fyle/delegated-accounts/delegated-accounts.page.html:26]: delegated--org-element-us-details

File: src/app/fyle/delegated-accounts/delegated-accounts.page.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        await this.loaderService.hideLoader();
+      })
+    ).subscribe(() => {
+      this.router.navigate(['/', 'enterprise', 'my_dashboard']);
+    });
+  }
+
+  ngOnInit() {
+
+    const switchToOwn = this.activatedRoute.snapshot.params.switchToOwn && JSON.parse(this.activatedRoute.snapshot.params.switchToOwn);
```
Comment (Julias0) [inline src/app/fyle/delegated-accounts/delegated-accounts.page.ts:42]: Why is parsing required here?

Comment (Julias0) [inline src/app/fyle/delegated-accounts/delegated-accounts.page.ts:42]: mention with a comment why parsing is required here

**Review by Julias0 — CHANGES_REQUESTED** [review]
Please fix

File: src/app/fyle/delegated-accounts/delegated-accounts.page.ts (hunk 4 above)
Comment (madhavmansuriya40) [reply src/app/fyle/delegated-accounts/delegated-accounts.page.ts:42]: actually, this is not required will remove it

**Review by Julias0 — APPROVED** [review]

**Pull request closed without merging**